src/proc/colorizer.cpp

```diff
@@ -156,6 +156,16 @@ namespace librealsense
         auto max_opt = std::make_shared<ptr_option<float>>(0.f, 16.f, 0.1f, 6.f, &_max, "Max range in meters");
         register_option(RS2_OPTION_MAX_DISTANCE, max_opt);
```

Comment:
Can you please try to remove the setting of the original options, since we think they're going to be overriden below?